tests/NpgsqlFSharpTests.fs

```diff
@@ -78,7 +78,7 @@ let buildDatabaseConnection handleInfinity : ThrowawayDatabase =
         |> Sql.port 5432
         |> Sql.username "postgres"
         |> Sql.password databasePassword
-        |> Sql.convertInfinityDateTime handleInfinity
+        //|> Sql.convertInfinityDateTime handleInfinity
```

Comment:
Why is this commented out?

Reply:
According to: [https://www.npgsql.org/doc/types/datetime.html](Npgsql documentation):

Note: in versions prior to 6.0, the connection string parameter Convert Infinity DateTime could be used to opt into these infinity conversions. That connection string parameter has been removed.

Reply:
In that case, any chance you could also remove the configuration option entirely from `ConnectionStringBuilder`?

Reply:
Excellent idea. Done!